Sources/ComposableArchitecture/Effects/Cancellation.swift

```diff
@@ -215,8 +215,15 @@ extension Task where Success == Never, Failure == Never {
   }
 }
 
-@_spi(Internals) public var _cancellationCancellables = CancellablesCollection()
-private let _cancellablesLock = NSRecursiveLock()
+#if swift(>=5.10)
+  @_spi(Internals)
+  nonisolated(unsafe)
+  public var _cancellationCancellables = CancellablesCollection()
+  private let _cancellablesLock = NSRecursiveLock()
```

Comment:
since we do our own synchronization of these variables we can turn off isolation checking. of course that means the responsibility is on us to make sure everything is correct, but luckily all of this goes away somewhat soon!